### PR TITLE
FIX: Raw.filter() picks=None include seeg

### PR DESCRIPTION
**Pull request by kingjr**
The previous raw.filter did not include seeg channels by default, and I didn't find any test of the picking behavior.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Travis is sad at you because dict comprehension doesn't work on Py26. You can instead do `dict((key, val) for key, val in zip(whatever))`.

File: doc/whats_new.rst [hunk 1]
```diff
@@ -86,6 +86,8 @@ API
 
     - To unify and extend the behavior of :func:`mne.comupute_raw_covariance` relative to :func:`mne.compute_covariance`, the default parameter ``tstep=0.2`` now discards any epochs at the end of the :class:`mne.io.Raw` instance that are not the full ``tstep`` duration. This will slighly change the computation of :func:`mne.compute_raw_covaraince`, but should only potentially have a big impact if the :class:`mne.io.Raw` instance is short relative to ``tstep`` and the last, too short (now discarded) epoch contained data inconsistent with the epochs that preceded it.
 
+    - The default `picks=None` in :func:`mne.io.raw.filter` nows picks eeg, meg and seeg channels, by `Jean-Remi King`_
```
Comment (Eric89GXL) [inline doc/whats_new.rst]: double-backticks on `picks=None` so it gets rendered as code

Comment (Eric89GXL) [inline doc/whats_new.rst]: and it probably needs to be `mne.io.Raw.filter`, case sensitivity of the class name

File: mne/io/fiff/tests/test_raw_fiff.py [hunk 2]
```diff
@@ -771,6 +771,32 @@ def test_filter():
...
+
+    # -- Deal with meg mag grad exception
+    ch_types = ('misc', 'stim', 'meg', 'eeg', 'seeg')
+
+    # -- Filter data channels
+    for ch_type in ('mag', 'grad', 'eeg', 'seeg'):
+        picks = {ch: ch == ch_type for ch in ch_types}
+        picks['meg'] = ch_type if ch_type in ('mag', 'grad') else False
+        raw_ = raw.pick_types(copy=True, **picks)
+        with warnings.catch_warnings(record=True) as w:
```
Comment (Eric89GXL) [inline mne/io/fiff/tests/test_raw_fiff.py]: make a comment here saying what the warning is, it's not obvious from the code or the assertion check

**Comment by Eric89GXL** [comment]
Otherwise LGTM

**Comment by agramfort** [comment]
close in favor in #2950

**Pull request closed without merging**